### PR TITLE
Fixes a few runtimes

### PR DESCRIPTION
**Pull request by SapphicOverload**
# Document the changes in your pull request

There are a few runtimes that always show up when I run a private server. This PR fixes some of them.

# Changelog

:cl:  
bugfix: fixes some runtimes in _component.dm, mood.dm, and inventory.dm
/:cl:

### CODE REVIEW

File: code/modules/mob/inventory.dm [hunk 1]
```diff
@@ -298,6 +298,8 @@
 //for when you want the item to end up on the ground
 //will force move the item to the ground and call the turf's Entered
 /mob/proc/dropItemToGround(obj/item/I, force = FALSE, silent = FALSE)
+	if(!I)
```
Comment (redmoogle) [inline code/modules/mob/inventory.dm]: #15727

**Comment by ToasterBiome** [comment]
redmoogle's solution to the dismemberment is better because we should be figuring out why we're putting null into things instead of fixing behavior when things are null

revert that line and the rest looks good, i wonder what could be causing the mood one

**Comment by JamieD1** [comment]
Can you make sure this works with moogles

**Comment by redmoogle** [comment]
> Can you make sure this works with moogles

Jamie they undid that part of the change

**Review by Bibby0110 — APPROVED** [review]

**Pull request merged**